runelite-client/src/main/scripts/PrivateChatBuilder.rs2asm

```diff
@@ -0,0 +1,902 @@
+.id                 84
```

Comment:
`PrivateChatBuilder` doesn't really make much sense because it handles all message types in the chat box.